.github/workflows/test-integrations-unreleased.yml

```diff
@@ -0,0 +1,55 @@
+name: Integration with Unreleased OpenSearch
+
+on:
+  push:
+    branches:
+      - "main"
+  pull_request:
+    branches:
+      - "main"
```

Comment:
Should build on all branches, no?

Reply:
Yes, updated it to all push and pull request.